docs/build_ios.md

```diff
@@ -0,0 +1,155 @@
+```
+
+## Step 3: Create toolchain environment
+After adding kivy-ios to your venv (I called mine mindset), create another diretory that will store the Xcode builds from the toolchain software package.
+
+Navigate into KivyBuilds (name of directory I am using to store Xcode project and the toolchain environment).
+
+A file structure I use for this work is
+```
+_environments/
+    mindset/
+KivyBuilds/
+    # toolchain environment will go here
+poc-mobil-python/
+```
+
+
+From inside KivyBuilds/ create the toolchain environemnt
+```
+toolchain build python3 kivy pillow libffi ffpyplayer 
+```
+This takes about 20 minutes with a good internet connection.
+
+```
+toolchain pip install kivymd kaki watchdog event_bus 
+```
+
+### Key Note on toolchain environment
+toolchain commands are particlar to the folder where toolchain packages are installed. The gif below demonstrates how your toolchain environment works. Once inside the folder where Step 3 is executed (i.e. KivyBuilds in the file structure example), if you do `toolchain status` you will see all the packages installed. However, if you naviagate outside of that folder toolchain status will return no install pacakges.
+
+<img src="../stores_presence/ios_build/toolchain_env_minus5.gif"/> 
+
+### Background on toolchain/ kivy-ios
+toolchain is both the (1) command and (2) pacakge/suite/collection of software availible in kivy-ios package. toolchain suite of software will will convert the python project (poc-mobile-python) to an Xcode project.
+
+## Step 4: Create Xcode proejct
+Again from inside your KivyBuilds/ do the following command:
+```
+toolchain create openmindset /Users/nick/Documents/poc-mobile-python
+```
+Here openmindset could be anything and the last argument is the path to the Python Kivy project.
+
+## Step 5: Signin to Xcode
+
+Sign in with your [Apple Developer account](https://developer.apple.com/programs/)
+<img src="../stores_presence/ios_build/XcodeSignIn.gif" /> 
+
+## Step 6: Build to simulator or iPhone
+Select device and press the play button
+
+## Build to TestFlight
+1. From inside Xcode select Product menu > Archive
+    - This creates a file that will be submitted to App Store Connects
+2. After Archive (.xcarchive file) is created select the file from Oragnizer and click "Distribute App".
+    - This can also be accessed by Window menu > Organizer
+3. There will be a automated process of verifying the package meets Apple requirements
+    - one issue we have had is a binary that is not accepted from the toolchain environment. The remedy to this is just delete the file and the package will get accepted.
+        - File to delete: KivyBuilds/dist/root/python3/lib/python3.10/site-packages/_watchdow_fsevents.cpython-310-darwin.so
+### Other iOS stuff
+Note:
+- Contributors are welcome to do this one, collected below some beginning instructions with some issues and solutions.
+- Rosetta 2 terminal not required for arm64 machines when using kivy-ios==2.2.1
```

Comment:
```suggestion
- Rosetta 2 terminal:
  - not required for arm64 machines when using kivy-ios==2.2.1
  - maybe required if you want to deploy on an `iOS iPhone Simulator`
```

Reply:
I made a mistake here it should be kivy-ios==2023.8.24. Should I accept then make change or is there a better way of making the correction before committing?